modules/kmm-signing-a-prebuilt-driver-container.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * hardware_enablement/kmm-kernel-module-management.adoc
+
+:_content-type: PROCEDURE
+[id="kmm-signing-a-prebuilt-driver-container_{context}"]
+= Signing a pre-built driver container
+
+Use this procedure if you have a pre-built image, such as an image distributed by a hardware vendor
+or is already built elsewhere.
+
+The YAML file shown below adds the public/private key-pair as secrets with the required key names - `key` for the private key, `cert` for the public key. It will then pull down the `unsignedImage` image, open it, sign the kernel modules listed in `filesToSign`, add them back and push the resulting image as `containerImage`.
```

Comment:
To avoid future tense:

> The cluster then pulls down the `unsignedImage` image, opens it, signs the kernel modules listed in `filesToSign`, adds them back, and pushes the resulting image as `containerImage`.

Or maybe rewrite as a list of some sort.

Reply:
changed.